pkgs/development/python-modules/pylatex/default.nix

```diff
@@ -0,0 +1,37 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, ordered-set
+}:
+
+buildPythonPackage rec {
+  pname = "pylatex";
+  version = "1.3.4";
```

Comment:
FYI: 1.4.2 has been released. https://github.com/JelteF/PyLaTeX/tree/v1.4.2

It does not need to be worked on by this PR, but may be worthwhile as python 3.12 will be coming to mainstream nixpkgs python in the near future.

Reply:
I changed that as well.